reactNativeProject/src/screens/DetailsBooks/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { View, Text } from "react-native";
+
+import styles from "./styles";
+
+class DeatailsBooks extends Component {
+  static navigationOptions = ({ navigation }) => {
+    return {
+      title: navigation.getParam("title")
+    };
+  };
+  render() {
+    const author = this.props.navigation.getParam("author", "NO-author");
```

Comment:
Remove all the default values like "NO-author". All the books have those. This applies for all these values